app/experimenter/experiments/models.py

```diff
@@ -196,6 +198,7 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         null=True,
     )
     takeaways_summary = models.TextField(blank=True, null=True)
+    updated = models.DateTimeField(auto_now=True)
```

Comment:
Oh can you please follow the same naming pattern as the other cached dates below ie `_updated_date` 🙏